thirdeye-ui/src/app/components/subscription-group-wizard/subscription-group-details/recipient-details/groups-editor/slack/slack.component.tsx

```diff
@@ -158,6 +241,100 @@ export const Slack: FunctionComponent<SlackProps> = ({
                         }
                         label={t("label.slack-url")}
                     />
+                    <InputSection
+                        inputComponent={
+                            <TextField
+                                fullWidth
+                                error={Boolean(
+                                    errors && errors.textConfiguration?.owner
+                                )}
+                                helperText={
+                                    errors &&
+                                    errors.textConfiguration?.owner &&
+                                    errors.textConfiguration?.owner?.message
```

Comment:
Not a problem, but this can be condensed to simply `errors?.textConfiguration?.owner?.message`.